packages/calcite-components/src/components/fab/fab.scss

```diff
@@ -1,17 +1,34 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-fab-shadow: Specifies the shadow of the component.
+ * @prop --calcite-fab-shadow-hover: Specifies the shadow of the component when in hover state.
+ * @prop --calcite-fab-shadow-active: Specifies the shadow of the component when in active state.
+ * @prop --calcite-fab-offset-shadow: Specifies the offset shadow of the component.
```

Comment:
Remove these. After looking it over again, I don't think fab has any of it's own tokens. We should direct customers to the button component for a list of themeable variables.

Reply:
Yeah I was just starting to realize this and was going to ask about it but then found these comments.

Reply:
It will be really hard for us to document the component-tokens if they aren't available on the component itself. 

Even if --calcite-fab-shadow just sets --calcite-button-shadow internally, I think that having a set of `--calcite-fab-x` would be more expected DX / would be able to show on the documentation site.